packages/block-library/src/comments-query-loop/edit.js

```diff
@@ -3,47 +3,61 @@
  */
 import {
 	BlockControls,
-	InspectorControls,
 	useBlockProps,
 	useInnerBlocksProps,
 } from '@wordpress/block-editor';
-import { SelectControl } from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import QueryToolbar from './toolbar';
+import CommentsToolbar from './edit/toolbar';
```

Comment:
Since you've created this new file `edit/toolbar` we should remove the old one in the root folder, no?